packaging/docker/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM centos:7
+LABEL maintainer="tdockendorf@osc.edu; johrstrom@osc.edu"
+
+# setup the ondemand repositories
+COPY files/ondemand-web.repo /etc/yum.repos.d/ondemand-web.repo
+COPY files/ondemand-ci-web.repo /etc/yum.repos.d/ondemand-ci-web.repo
+RUN curl https://yum.osc.edu/ondemand/RPM-GPG-KEY-ondemand -o /etc/pki/rpm-gpg/RPM-GPG-KEY-ondemand
+RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-ondemand
+
+# update
+RUN yum install -y centos-release-scl 
+RUN yum -y update
+
+# install all the dependencies
+RUN yum install -y \
+        file \
+        lsof \
+        sqlite-devel \
+        sudo \
+        httpd24 \
+        rh-ruby25 \
+        rh-nodejs10 \
+        ondemand-passenger-6.0.4-2.el7.x86_64 \
```

Comment:
Can package versions be parameterized using build-time environment variables?

Reply:
Yes, Docker uses `ARG` but there's nothing calling this. No entry point yet, so you can straight run `docker build -t ood:src .` instead of having to have a wrapper with all sorts of ARGs (like I did for writing a VERSION file)

Reply:
Seems like we should use https://github.com/OSC/ondemand/blob/f42ac1a807e3cf31aa6df098597eec8676c30128/packaging/ondemand.spec#L55-L61 dependencies instead of rh-ruby25 rh-nodejs10 directly.

Reply:
I wouldn't specify a version, just install latest.  We typically don't put something into repo unless it's something we intend to use or upgrade to.